log-backup: added an explaination to the log status (#20564)
pingcap/docs: br/br-pitr-guide.md

```
● Total 1 Tasks.
> #1 <
           name: pitr
         status: ● NORMAL
          start: 2022-05-13 11:09:40.7 +0800
            end: 2035-01-01 00:00:00 +0800
        storage: s3://backup-101/log-backup
    speed(est.): 0.00 ops/s
checkpoint[global]: 2022-05-13 11:31:47.2 +0800; gap=4m53s
```

The fields are explained as follows:

- `name`: the name of the log backup task.
- `status`: the status of the log backup task, including `NORMAL`, `PAUSED`, and `ERROR`.
- `start`: the start timestamp of the log backup task.
- `end`: the end timestamp of the log backup task. Currently, this field does not take effect.
- `storage`: the URI of the external storage for the log backup.

> - `speed(est.)`: the current data transfer rate of the log backup. This value is estimated based on recent samples over a few seconds. For more accurate traffic statistics, you can check the `Log Backup` row in the `tikv-details` dashboard at Grafana.

Reviewer: ```suggestion
- `speed(est.)`: the current data transfer rate of the log backup. This value is estimated based on traffic samples taken in the past few seconds. For more accurate traffic statistics, you can check the `Log Backup` row in the **[TiKV-Details](/grafana-tikv-dashboard.md#tikv-details-dashboard)** dashboard at Grafana.
```

> - `speed(est.)`: the current data transfer rate of the log backup. This value is estimated based on recent samples over a few seconds. For more accurate traffic statistics, you can check the `Log Backup` section under the `tikv-details` monitoring dashboard.

Reviewer 2: Tweaking some terms.

```suggestion
- `speed(est.)`: the current data transfer rate of the log backup. This value is estimated based on recent samples over a few seconds. For more accurate traffic statistics, you can check the `Log Backup` row in the `tikv-details` dashboard at Grafana.
```

> - `name`: the name of the log backup task.
> - `status`: the status of the log backup task, including `NORMAL`, `PAUSED`, and `ERROR`.
> - `start`: the start timestamp of the log backup task.
> - `end`: the end timestamp of the log backup task. Currently, this field does not take effect.
> - `storage`: the URI of the external storage for the log backup.
> - `speed(est.)`: the current data transfer rate of the log backup. This value is estimated based on recent samples over a few seconds. For more accurate traffic statistics, you can check the `Log Backup` row in the `tikv-details` dashboard at Grafana.
> - `checkpoint[global]`: the current progress of the log backup. You can use PITR to restore to a point in time before this timestamp.

Reviewer: ```suggestion
- `name`: the name of the log backup task.
- `status`: the status of the log backup task, including `NORMAL`, `PAUSED`, and `ERROR`.
- `start`: the start timestamp of the log backup task.
- `end`: the end timestamp of the log backup task. Currently, this field does not take effect.
- `storage`: the URI of the external storage for the log backup.
- `speed(est.)`: the current data transfer rate of the log backup. This value is estimated based on traffic samples taken in the past few seconds. For more accurate traffic statistics, you can check the `Log Backup` row in the **[TiKV-Details](/grafana-tikv-dashboard.md#tikv-details-dashboard)** dashboard at Grafana.
- `checkpoint[global]`: the current progress of the log backup. You can use PITR to restore to a point in time before this timestamp.
```

If the log backup task is paused, the `log status` command outputs additional fields to display the details of the pause. For example:

```
● Total 1 Tasks.
> #1 <
              name: pitr
            status: ○ ERROR
               <......>
        pause-time: 2025-03-14T14:35:06+08:00
    pause-operator: atelier.local
pause-operator-pid: 64618
     pause-payload: The checkpoint is at 2025-03-14T14:34:54+08:00, now it is 2025-03-14T14:35:06+08:00, the lag is too huge (11.956113s) hence pause the task to avoid impaction to the cluster
```

The meanings of these additional fields are as follows:

- `pause-time`: the time when the pause operation is executed.
- `pause-operator`: the hostname of the machine that executes the pause operation.
- `pause-operator-pid`: the PID of the process that executes the pause operation.
- `pause-payload`: additional information attached when the task is paused.

> If the error occurs on TiKV, you might also see additional error reports from TiKV:

Reviewer: ```suggestion
If the pause is due to an error in TiKV, you might also see additional error reports from TiKV:
```

> - `error[store=*]`: the error code of the error that occurs on TiKV.
> - `error-happen-at[store=*]`: the time when the error occurs on TiKV.
> - `error-message[store=*]`: the message of the error that occurs on TiKV.

Reviewer: ```suggestion
- `error[store=*]`: the error code on TiKV.
- `error-happen-at[store=*]`: the time when the error occurs on TiKV.
- `error-message[store=*]`: the error message on TiKV.
```
